tests/fixtures/ntfs/trips.txt

```diff
@@ -1,4 +1,6 @@
 route_id,service_id,trip_id,company_id,physical_mode_id,dataset_id
+M1F,Week,M1FF,TGC,Metro,TGDS
+M1B,Week,M1BF,TGC,Metro,TGDS
```

Comment:
you could add frequencies on existing trips instead of creating others

Reply:
OK, I changed it to modify the existing trips. I also changed the Stop Times since a best practice in the case of `frequencies.txt` is to start from midnight.